### PR TITLE
simplify loading call

### PR DESCRIPTION
**Pull request by mjburton11**

### CODE REVIEW

**Comment by mjburton11** [comment]
test models please

**Comment by mjburton11** [comment]
test models please

File: gpkitmodels/GP/aircraft/wing/sparloading.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def new_qbarFun(self, c):
         barc = self.static["\\bar{c}"]
         return [f(c) for f in self.static.substitutions[barc]]
 
-    def setup(self, static, Wcent):
+    def setup(self, static, Wcent=None):
```
Comment (bqpd) [inline gpkitmodels/GP/aircraft/wing/sparloading.py]: Why is Wcent even an argument?

File: gpkitmodels/GP/aircraft/wing/gustloading.py [hunk 2]
```diff
@@ -32,9 +35,12 @@ def setup(self, static, Wcent, Wwing, V, CL):
 
         constraints = [
             # fit for arctan from 0 to 1, RMS = 0.044
-            FitCS(df, agust, [cosminus1*vgust/V]),
+            V == v,
+            CL == cl,
+            Ww == Wwing,
```
Comment (bqpd) [inline gpkitmodels/GP/aircraft/wing/gustloading.py]: why are these necessary?

Comment (mjburton11) [reply gpkitmodels/GP/aircraft/wing/gustloading.py]: for backwards compatibility so I don't break jho and gassolar

Comment (mjburton11) [reply gpkitmodels/GP/aircraft/wing/gustloading.py]: I'll fix gassolar and jho once this is done

Comment (bqpd) [reply gpkitmodels/GP/aircraft/wing/gustloading.py]: I'd be tempted to make their usage of this more hacky (repeating arguments or w/e) rather than have confusing code in gplibrary...
